go-controller/hybrid-overlay/pkg/types/types.go

```diff
@@ -11,9 +11,13 @@ const (
 	HybridOverlayNodeSubnet = HybridOverlayAnnotationBase + "node-subnet"
 	// HybridOverlayDRMAC holds the MAC address of the Distributed Router/gateway
 	HybridOverlayDRMAC = HybridOverlayAnnotationBase + "distributed-router-gateway-mac"
+	// HybridOverlayExternalGW holds the IP address of the external gateway to route default traffic
+	HybridOverlayExternalGw = HybridOverlayAnnotationBase + "external-gw"
 
 	// HybridOverlayVNI is the VNI for VXLAN tunnels between nodes/endpoints
 	HybridOverlayVNI = 4097
+	// HybridOverlayExternalGWVNI is the VNI for VXLAN tunnels between nodes and the external gateway
```

Comment:
@rcarrillocruz I thought we decided to use the same VNID for now... maybe I'm wrong?

Reply:
We agreed to use 4098, as down the line we can expect users/customers to use both Windows and external GW on hybrid overlay, thus the separate VNIDs.

Reply:
@rcarrillocruz simpler (especially if we need to customize the VNID) to use the same VNID for now. There's no conflict with Windows and external GW because you'll never run the external GW on a Windows node and thus they won't fight over the VTEP.